Bounding_volumes/doc/Bounding_volumes/CGAL/Min_sphere_of_spheres_d_traits_d.h

```diff
@@ -38,7 +38,7 @@ class Min_sphere_of_spheres_d_traits_d {
 /*!
 is the constant `Dim`. 
 */ 
-typedef unspecified_type D; 
+static const int D;
```

Comment:
Probably "is the constant Dim::value" now? Maybe say somewhere in this file that the template parameter Dim cannot be Dynamic_dimension_tag, it has to be `Dimension_tag<D>`?